### ISSUE TITLE
Movie Recommendation System 

### ISSUE THREAD

**Issue by Jogesh-Mishra** [Jayshah6699/datascience-mashup#19]
The model takes input data of various users, the movies they have watched and if they like it or not. Later on, the model is trained on that data by implementing Real Boltzmann Machine using Pytorch to predict the movies in the test set that the concerned user may like to watch.

**Comment by Jayshah6699** [comment]
@Jogesh-Mishra kindly provide the description

**Comment by Jogesh-Mishra** [comment]
> @Jogesh-Mishra kindly provide the description

Done.

**Comment by Amit366** [comment]
Sir I would like to work on this issue
